app/src/main/java/org/schabi/newpipe/player/MainVideoPlayer.java

```diff
@@ -838,26 +840,40 @@ public boolean onSingleTapConfirmed(MotionEvent e) {
             return true;
         }
 
-        private final boolean isPlayerGestureEnabled = PlayerHelper.isPlayerGestureEnabled(getApplicationContext());
+        public boolean isPlayerGestureEnabled = true; //PlayerHelper.isPlayerGestureEnabled(getApplicationContext());
 
         private final float stepsBrightness = 15, stepBrightness = (1f / stepsBrightness), minBrightness = .01f;
         private float currentBrightness = .5f;
 
-        private int currentVolume, maxVolume = playerImpl.getAudioReactor().getMaxVolume();
+        private int currentVolume, maxVolume = 100;//playerImpl.getAudioReactor().getMaxVolume();
         private final float stepsVolume = 15, stepVolume = (float) Math.ceil(maxVolume / stepsVolume), minVolume = 0;
 
         private final String brightnessUnicode = new String(Character.toChars(0x2600));
         private final String volumeUnicode = new String(Character.toChars(0x1F508));
 
         private final int MOVEMENT_THRESHOLD = 40;
         private final int eventsThreshold = 8;
-        private boolean triggered = false;
+        public boolean triggered = true;
+        public int currentState = 127;
+        public int getWidth = 500;
+        public int getVolume = 50;
+        public int getVisibility = 0;
         private int eventsNum;
 
         // TODO: Improve video gesture controls
-        @Override
-        public boolean onScroll(MotionEvent e1, MotionEvent e2, float distanceX, float distanceY) {
-            if (!isPlayerGestureEnabled) return false;
+        // @Override - Kind of had to remove this...
+        public boolean onScroll(MotionEvent e1, MotionEvent e2, float distanceX, float distanceY, CodeCoverage... codeCoverage) {
```

Comment:
Please add requirements for onScroll() as a comment above it. You can see how I and Emil have done it.